addons/base_partner_address/models/res_partner.py

```diff
@@ -130,8 +124,6 @@ def create(self, vals):
         if vals.get('is_company', False):
```

Comment:
@kaerdsar I checked some code parts: Can you remove the lines 124 - 126, it was a project specific change. In addition you can remove the name_get() function completely, it was a project specific change too.